data/root/root.file_list

```diff
@@ -448,12 +448,7 @@ if arch ne 's390' && arch ne 's390x'
   ?xf86-input-vmmouse:
   xf86-input-void:
   xf86-input-wacom:
-  xf86-video-ati: nodeps
-  xf86-video-cirrus:
```

Comment:
https://linux.die.net/man/1/qemu-kvm mentions "Cirrus CLGD 5446 PCI VGA card", will KVM work correctly without that driver?